commands/databases.go

```diff
@@ -0,0 +1,851 @@
+	if err != nil {
+		return nil, err
+	}
+	r.NumNodes = numNodes
+
+	size, err := c.Doit.GetString(c.NS, doctl.ArgSizeSlug)
+	if err != nil {
+		return nil, err
+	}
+	r.SizeSlug = size
+
+	return r, nil
+}
+
+// RunDatabaseMigrate migrates a database cluster to a new region
+func RunDatabaseMigrate(c *CmdConfig) error {
+	if len(c.Args) == 0 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	id := c.Args[0]
+
+	r, err := buildDatabaseMigrateRequestFromArgs(c)
+	if err != nil {
+		return err
+	}
+
+	return c.Databases().Migrate(id, r)
+}
+
+func buildDatabaseMigrateRequestFromArgs(c *CmdConfig) (*godo.DatabaseMigrateRequest, error) {
+	r := &godo.DatabaseMigrateRequest{}
+
+	region, err := c.Doit.GetString(c.NS, doctl.ArgRegionSlug)
+	if err != nil {
+		return nil, err
+	}
+	r.Region = region
+
+	return r, nil
+}
+
+func databaseMaintenanceWindow() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "maintenance-window",
+			Aliases: []string{"maintenance", "mw", "main"},
+			Short:   "maintenance window commands",
+			Long:    "maintenance is used to access maintenance window commands for a database cluster",
+		},
+	}
+
+	CmdBuilder(cmd, RunDatabaseMaintenanceGet, "get <database-id>", "get maintenance window info", Writer, aliasOpt("g"))
+
+	cmdDatabaseCreate := CmdBuilder(cmd, RunDatabaseMaintenanceUpdate, "update <database-id>", "update maintenance window", Writer,
+		aliasOpt("u"))
+	AddStringFlag(cmdDatabaseCreate, doctl.ArgDatabaseMaintenanceDay, "", "", "new maintenance window day", requiredOpt())
+	AddStringFlag(cmdDatabaseCreate, doctl.ArgDatabaseMaintenanceHour, "", "", "new maintenance window hour", requiredOpt())
+
+	return cmd
+}
+
+// Database Maintenance Window
+
+// RunDatabaseMaintenanceGet retrieves the maintenance window info for a database cluster
+func RunDatabaseMaintenanceGet(c *CmdConfig) error {
+	if len(c.Args) == 0 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	id := c.Args[0]
+
+	window, err := c.Databases().GetMaintenance(id)
+	if err != nil {
+		return err
+	}
+
+	return displayDatabaseMaintenanceWindow(c, *window)
+}
+
+func displayDatabaseMaintenanceWindow(c *CmdConfig, mw do.DatabaseMaintenanceWindow) error {
+	item := &displayers.DatabaseMaintenanceWindow{DatabaseMaintenanceWindow: mw}
+	return c.Display(item)
+}
+
+// RunDatabaseMaintenanceUpdate updates the maintenance window info for a database cluster
+func RunDatabaseMaintenanceUpdate(c *CmdConfig) error {
+	if len(c.Args) == 0 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	id := c.Args[0]
+	r, err := buildDatabaseUpdateMaintenanceRequestFromArgs(c)
+	if err != nil {
+		return err
+	}
+
+	return c.Databases().UpdateMaintenance(id, r)
+}
+
+func buildDatabaseUpdateMaintenanceRequestFromArgs(c *CmdConfig) (*godo.DatabaseUpdateMaintenanceRequest, error) {
+	r := &godo.DatabaseUpdateMaintenanceRequest{}
+
+	day, err := c.Doit.GetString(c.NS, doctl.ArgDatabaseMaintenanceDay)
+	if err != nil {
+		return nil, err
+	}
+	r.Day = day
```

Comment:
```suggestion
	r.Day = strings.ToLower(day)
```
Since this needs to be lowercase, let's do that for the user.

```
$ ./builds/doctl databases maintenance update 28ddd061-0822-47bd-9c12-d2f46747cc6c --day Monday --hour 18:00
# [snipping usage info]
Error: PUT https://api.digitalocean.com/v2/databases/28ddd061-0822-47bd-9c12-d2f46747cc6c/maintenance: 500 Server Error
```

Might want to adjust this on the backend as well.